group_vars/kube-masters

```diff
@@ -0,0 +1 @@
+kube_role: master
```

Comment:
I think this file can be removed in favour of a different approach (see my other comments).